### ISSUE TITLE
tflint returning exit code 2 - creating bash false negatives

### ISSUE THREAD

**Issue by stevieschatz** [terraform-linters/tflint#1647]
### Summary

Due to your [documentation](https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/config.md#force), `tflint` is returning exit code 2 when there are warnings. 

This is breaking our pipeline as bash interprets non-zero exit codes as an indication for failure. [bash-failure-codes](https://www.cyberciti.biz/faq/linux-bash-exit-status-set-exit-statusin-bash/#:~:text=For%20the%20bash%20shell's%20purposes,the%20return%20status%20is%20126.)
For example, this means we have to add in our github actions tflint step `continue-on-error` which leads to us missing "real" linting errors.

### Command

tflint -f compact

### Terraform Configuration

```terraform
No special configuration.
```

### TFLint Configuration

```HCL
`tflint -f compact` default configuration used.
```

### Output

```Shell
15:44:26 [DEBUG] go-plugin@v1.4.8/client.go:603: starting plugin: path=/runner/_work/_temp/6c4fc45e-8f9f-478e-963e-1085989a4ee0/tflint args=["/runner/_work/_temp/6c4fc45e-8f9f-478e-963e-1085989a4ee0/tflint", "--act-as-bundled-plugin"]
15:44:26 [DEBUG] go-plugin@v1.4.8/client.go:611: plugin started: path=/runner/_work/_temp/6c4fc45e-8f9f-478e-963e-1085989a4ee0/tflint pid=525
15:44:26 [DEBUG] go-plugin@v1.4.8/client.go:706: waiting for RPC address: path=/runner/_work/_temp/6c4fc45e-8f9f-478e-963e-1085989a4ee0/tflint
15:44:26 [DEBUG] go-plugin@v1.4.8/client.go:1045: tflint: 15:44:26 [DEBUG] go-plugin@v1.4.8/server.go:401: plugin address: network=unix address=/tmp/plugin3308962084
15:44:26 [DEBUG] go-plugin@v1.4.8/client.go:751: using plugin: version=11
15:44:26 [DEBUG] host2plugin/client.go:124: starting host-side gRPC server
15:44:26 [DEBUG] go-plugin@v1.4.8/grpc_stdio.go:139: stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
15:44:26 [INFO]  go-plugin@v1.4.8/client.go:664: plugin process exited: path=/runner/_work/_temp/6c4fc45e-8f9f-478e-963e-1085989a4ee0/tflint pid=525
15:44:26 [DEBUG] go-plugin@v1.4.8/client.go:469: plugin exited
5 issue(s) found:

Warning: main.tf:18:41: Warning - Interpolation-only expressions are deprecated in Terraform v0.12.14 (terraform_deprecated_interpolation)
Warning: variables.tf:48:1: Warning - `environment` variable has no type (terraform_typed_variables)
Warning: variables.tf:52:1: Warning - `location` variable has no type (terraform_typed_variables)
Warning: variables.tf:61:1: Warning - `blob_versioning_enabled` variable has no type (terraform_typed_variables)
Warning: variables.tf:66:1: Warning - variable "key_vault_sas_token_content_type" is declared but not used (terraform_unused_declarations)
Error: Process completed with exit code 2.
```

### TFLint Version

v0.44.1

### Terraform Version

1.3.7

### Operating System

- [X] Linux
- [X] macOS
- [ ] Windows

**Comment by bendrucker** [comment]
As mentioned, this is the documented/expected behavior. See #1556 for the existing related feature request.
